lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -1856,6 +1856,9 @@ namespace Js
             {
                 if (descriptor->IsAccessor && !(attributes & PropertyLetConstGlobal))
                 {
+#if DEBUG
+                    Var ctor = JavascriptOperators::GetProperty(instance, PropertyIds::constructor, scriptContext);
```

Comment:
How does this ctor work below if it's defined in `#if DEBUG`?

Reply:
It's only used in the assert